packages/documentation/src/pages/getting-started/common-tasks/releases.mdx

```diff
@@ -1,109 +0,0 @@
-# Deployment Process
-
-Code goes through several steps to get to production. This document describes this process. It should also be noted that this is the same flow for both content and code changes.
-
-## Automated Deployment Schedule
-
-| Application | Branch | Changes in by | Deployment Start | Release History |
-| --- | --- | --- | --- | --- |
-| vets-website | master | 2:00pm ET M-F | 3:00pm ET M-F| https://github.com/department-of-veterans-affairs/vets-website/releases |
-| vets-api | master | 2:00pm ET M-F | 3:00pm ET M-F | https://github.com/department-of-veterans-affairs/vets-api/releases |
-
-## Summary
-
-For feature and hotfix development (code changes and content changes), the process looks like:
-
-1. Dev/Content: Create feature branch from `master` branch
-1. Dev/Content: Commit changes to feature branch
-1. Dev/Content: Feature branch merged to the `master` branch (via Pull Request)
-1. Automatic: Build run from `master` branch to create an artifact
-1. Automatic: Deploy newly created artifact to dev and staging
-1. Automatic: Create a release in GitHub from master, tag artifacts of that commit sha with release name
-1. Automatic: Deploy to production using artifacts
-
-## Process Details
-
-The first half of the steps of are:
```

Comment:
A lot of this was duplicated in the CI document. I consolidated the CI document and the deploy document into the three workflow documents.